concepts/0003-protocols/semver.md

```diff
@@ -1,93 +1,51 @@
-or protocol is expressed in the `semver` portion of its [identifying URI](
-uris.md).
+[Semver](https://semver.org) rules apply to protocols, with the version of a protocol is expressed
+in the `semver` portion of its [identifying URI](#message-type-and-protocol-identifier-uris).
+The ["ingredients"](#ingredients) of a protocol combine to form a
+[public API in the semver sense](https://semver.org/#spec-item-1). Core Aries protocols
+specify only major and minor elements in a version; the patch component is not used.
 
-Individual message types are versioned as part of a coherent protocol, which
-constitutes a [public API in the semver sense](https://semver.org/#spec-item-1).
-An individual message type can add new optional fields, or deprecate
-existing fields, [with only a change to its protocol's minor
-version](https://semver.org/#spec-item-7).
-Similarly, a protocol can add new message types (or [adopted
-ones](../../0000-template-protocol.md#adopted-messages)) with only a change
-to the minor version. It can announce deprecated fields. It can add additional
-semantics around optional decorators. These are all backwards-compatible
-changes, also requiring only a minor version update.
+The major and minor versions of protocols match semvar semantics:
 
-Changes that remove fields or message types, that make formerly optional
+- Clarification updates that do not change the "public API" of the protocol can be made
+with community support _without_ changing the minor version of the protocol.
+- An individual message type can add new optional fields, or deprecate
+existing fields, [with only a change to its protocol's minor
+version](https://semver.org/#spec-item-7). Similarly, a protocol can
+add new or deprecate message types with only a change to the minor version.
+- Changes that remove fields or message types, that make formerly optional
 things required, or that alter the state machine in incompatible
 ways, must result in an [increase of the major version of the protocol/primary
 message family](https://semver.org/#spec-item-8).
 
-Because protocol handling choices depend mainly on major and minor version
-numbers, protocol versions are often simplified to major.minor. However,
-more complex versions do have defined behavior, and should be handled
-correctly by agents.
-
-### Version Negotiation
-
-The semver portion of a [message type or protocol identifier URI](uris.md) is
-used to establish the version of a protocol that parties use during an
-interaction.
-
-#### Initiator
-
-Unless Alice's agent (the initiator of a protocol) knows from prior history
-that it should do something different, it should begin a protocol using the
-highest version number that it supports. For example, if A.1
-supports versions 2.0 through 2.2 of protocol X, it should use 2.2 as the
-version in the message type of its first message.
-
-#### Recipient Rules
-
-Agents for Bob (the recipient) should reject messages from protocols with major
```

Comment:
This section is important. It has come up on community calls and is not well understood. It should not be deleted.

Reply:
I found this section to be extremely confusing.  I think the rules list above cover it sufficiently.

Reply:
The rules I mentioned might be below :-)

Reply:
Added as "Semver Examples".